packages/web/components/Post/components/Avatar.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { IAvatar } from "../types";
+
+export const Avatar: React.FC<IAvatar> = React.memo(
+  ({ image, profileLink, height }) => {
```

Comment:
Skoro `height` jest opcjonalne, to tutaj można zdefiniować jego wartość domyslną np. `height=40`. Warunek w linii `12` będzie niepotrzebny